lib/ims/lti/models/lti_model.rb

```diff
@@ -4,8 +4,9 @@ class LTIModel
     LTI_VERSION_2P1 = 'LTI-2p1'.freeze
 
     def initialize(attributes = {})
-      self.attributes = attributes
       @ext_attributes = {}
+      @unknown_attributes = {}
```

Comment:
Is adding support for this a certification thing? I just want to have context for this change.